### ISSUE TITLE
Don't move to next article with toggle-article-read

### ISSUE THREAD

**Issue by sarmong** [newsboat/newsboat#2271]
Right now when I trigger `toggle-article-read`, selection moves to the next item. In my opinion, it is out of responsibilities of this function.

For me, it would be more convenient to move upwards, because I usually look at the articles from the bottom (older) to the top (newer).

Would be nice to have an option to change this behaviour to go up and not go at all.

**Comment by sarmong** [comment]
Not really, because there are hundreds of articles and I want to see them date asc

**Comment by Limero** [comment]
As someone who also prefers to read articles from the bottom, I think the optimal behavior for this function is to move to the next unread item in the list, but if there are no unread below, move to the next unread above. This way, you can easily go through a list of unread articles from either direction.

However, after this has been implemented: https://github.com/newsboat/newsboat/issues/1165
Being able to bind a single key to do both actions might be preferred.

**Comment by Jorenar** [comment]
Beside UX, I'd also add another, more consequential, motivation: if you have a binding `toggle-article-read; prev; foo`, while it works most of the time, if you execute it on the last item, action `foo` will be done on wrong position.
